templates/invitation.html

```diff
@@ -11,3 +11,5 @@
 
 <p>Many thanks<br />
 Animals in Science Regulation Unit</p>
+
+<p>(This is an automated email. Please do not reply.)</p>
```

Comment:
They can reply, it will go to animalscience@digital.homeoffice.gov.uk

Reply:
@pcuk1979 did you see my comment on this? Do we want to tell them they can't reply when actually they can?